default.nix

```diff
@@ -6,6 +6,5 @@ mkShell {
     (haskell.packages.ghc8107.ghcWithPackages (p: with p; [
       cabal-install
     ]))
-    ormolu # for Haskell formatting
```

Comment:
Why did this get binned? I think I need it for running Ormolu on my system.

Reply:
This is not the same version as the CI.